library/dokku_network_property.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# from ansible.module_utils.basic import *
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  network:
+    description:
+      - The name of the network to attach to
+    required: True
+    default: null
+    aliases: []
+  property:
+    description:
+      - The attach network property
```

Comment:
This is just a suggestion, perhaps not the best
```suggestion
      - The network property of the app to be modified
```